src/event/event_kevent.c

```diff
@@ -528,11 +534,17 @@ _dispatch_kevent_merge_muxed(dispatch_kevent_t ke)
 	}
 }
 
+#define DISPATCH_KEVENT_ERSATZ_EVFILT_USER_IDENT 0xfffffe00
```

Comment:
This is unfortunate to have to use timers with this ident to have to signal the manager thread on this platform - should we have a comment to explain the purpose here?

Also, can you clarify what ERSATZ is?

Reply:
Will bad things happen if a `DISPATCH_SOURCE_TYPE_INTERVAL` timer is created with this magic value as the interval? I think that's the only EVFILT_TIMER source that uses something other than a clockid as the ident.

Reply:
I don't think so. As I understand it/unless I'm missing something, the only other kevent with EVFILT_TIMER here has timer index masked with `DISPATCH_KEVENT_TIMEOUT_IDENT_MASK` in the ident (`_dispatch_event_loop_timer_program`).   `DISPATCH_KEVENT_ERSATZ_EVFILT_USER_IDENT` shouldn't conflict with `DISPATCH_KEVENT_TIMEOUT_IDENT_MASK`.

(maybe DISPATCH_KEVENT_ERSATZ_EVFILT_USER_IDENT might be best fully sign extended here. I'll try that while I'm adding the comment mentioned above.)